public/_templates/bulma/header.tpl

```diff
@@ -44,7 +44,7 @@
 					</div>
 				</div>
 				<div class="navbar-end">
-					<div class="navbar-item">
+					<div class="navbar-item is-expanded">
```

Comment:
Out of curiosity, why is the class needed? The styles you're adding in this PR make no reference to this class.

Reply:
`.is-expanded` does a `flex-grow: 1;` for those `nav-item`s which we need here. It comes from the used bulma framework.